lib/export/cimpl6Exporter.js

```diff
@@ -100,26 +100,26 @@ class Cimpl6Exporter {
     let ns = namespace.replace(/\./, '_');
 
     const hierarchyPath = `${filePath}/${ns}.txt`;
-//console.log("hierarchyPath = "+JSON.stringify(hierarchyPath));  
+//console.log("hierarchyPath = "+JSON.stringify(hierarchyPath));
     mkdirp.sync(hierarchyPath.substring(0, hierarchyPath.lastIndexOf('/')));
     fs.writeFileSync(hierarchyPath, formattedFile);
-//console.log("DONE WRITING FILE");  
+//console.log("DONE WRITING FILE");
   }
 
-  /*exportValueSetFileToNamespace(formattedFile, namespace, filePath) {
+  exportValueSetFileToNamespace(formattedFile, namespace, filePath) {
     let ns = namespace.replace(/\./, '_');
 
     const hierarchyPath = `${filePath}/${ns}_vs.txt`;
     mkdirp.sync(hierarchyPath.substring(0, hierarchyPath.lastIndexOf('/')));
     fs.writeFileSync(hierarchyPath, formattedFile);
-  }*/
+  }
 
-  /*exportMappingsFileToNamespace(formattedFile, namespace, filePath) {
+  exportMappingsFileToNamespace(formattedFile, namespace, filePath) {
```

Comment:
Same as `exportValueSetFileToNamespace` -- I think we should add a disclaimer in a comment.

Reply:
Yup, I'll comment it out again.